docs/gallery/python/2_average_expression.py

```diff
@@ -0,0 +1,299 @@
+# interest (e.g., *ACTB*) as values.
+# These coefficients, which range from -1 (perfect negative correlation) to 1
+# (perfect positive correlation), indicate how similar each gene's expression
+# pattern is to that of the target gene.
+#
+# For example, *LASP1* has the highest similarity to *ACTB* (0.483309),
+# suggesting potential co-regulation across cell types in the lung.
+
+# %%
+# Additionally, other than correlation, methods like euclidean and
+# manhattan are also available. Please refer to the `atlasapprox <https://atlasapprox.readthedocs.io/en/latest/index.html>`_
+# API documentation for details.
+
+# %%
+# Visualise the expression profile of these genes
+# -----------------------------------------------
+# To visualize the expression profiles of these genes, use a heatmap by
+# extracting gene names from ``similar_features.index`` and including the gene
+# of interest (e.g., *ACTB*) for comparison.
+#
+# Pass these as the ``features`` parameter to the ``average`` method, then plot
+# the results:
+
+# Combine ACTB and its similar features into a single list for better comparison
+gene_list = ['ACTB'] + list(similar_features.index)
+
+# Get average gene expression
+avg_similar_features = api.average(
+    organism='h_sapiens',
+    organ='lung',
+    features=gene_list
+)
+
+# Display the heatmap
+fig, ax = plt.subplots(figsize=(8, 6))
+heatmap = sns.heatmap(avg_similar_features,ax=ax)
+
+# Customize labels
+plt.title(f'Average gene expression across ACTB and its similar features')
+plt.xlabel("Cell type")
+plt.ylabel("Gene")
+
+fig.tight_layout()
+
+# %%
+# .. _marker-genes:
+#
+# Finding cell-type-specific markers in an organ
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# So far, the focus has been on known genes or those similar to a gene of interest.
+# But what if no specific gene comes to mind?
+# One way to begin is by exploring marker genes — genes that are highly and
+# specifically expressed in certain cell types — and then query their average
+# expression.
+#
+# For example, here's how to retrieve the top 10 marker genes for neutrophils in
+# the human lung:
+
+markers_in_lung_neutrophil = api.markers(
+    organism='h_sapiens',
+    organ='lung',
+    cell_type='neutrophil',
+    number=10
+)
+
+markers_in_lung_neutrophil
+
+# %%
+# Visualize expression profiles using a heatmap
+# ---------------------------------------------
+
+avg_gene_expr_markers = api.average(
+    organism='h_sapiens',
+    organ='lung',
+    features=markers_in_lung_neutrophil
+)
+
+fig, ax = plt.subplots(figsize=(8, 6))
+heatmap = sns.heatmap(avg_gene_expr_markers, ax=ax)
+
+# Set labels
+plt.title("Average expression of marker genes in lung neutrophil")
+plt.xlabel("Cell type")
+plt.ylabel("Gene")
+
+fig.tight_layout()
+
+# %%
+# The heatmap displays average expression levels of five neutrophil marker genes
+# across cell types. Most areas appear dark, indicating low expression (0-20 cptt)
```

Comment:
I don't think you explain what cptt is and why 20 cptt is low expression. Also 20 cptt is not low expression